website/docs: configuration: remove deprecated key for session storage location (#14431)
goauthentik/authentik: website/docs/security/cves/CVE-2025-29928.md

## Deletion of sessions did not revoke sessions when using database session storage

> ### ADDENDUM May 26, 2025: As of version 2025.4, the option to store sessions in cache has been removed; sessions are now exclusively stored in the database. See our [2025.4 release notes](../../releases/2025.4#sessions-are-now-stored-in-the-database) for more information.

Author: ```suggestion
### ADDENDUM May 30, 2025: As of version 2025.4, the option to store sessions in cache has been removed; sessions are now exclusively stored in the database. See our [2025.4 release notes](../../releases/2025.4#sessions-are-now-stored-in-the-database) for more information.
```

> ### ADDENDUM May 26, 2025: As of version 2025.4, the option to store sessions in cache has been removed; sessions are now exclusively stored in the database. See our [2025.4 release notes](https://version-2025-4.goauthentik.io/docs/releases/2025.4#sessions-are-now-stored-in-the-database) for more information.

Reviewer: ```suggestion
### ADDENDUM May 26, 2025: As of version 2025.4, the option to store sessions in cache has been removed; sessions are now exclusively stored in the database. See our [2025.4 release notes](../../releases/2025.4#sessions-are-now-stored-in-the-database) for more information.
```

> ### ADDENDUM May 8, 2025: As of version 2025.4, the option to store sessions in cache has been removed, and sessions are now exclusively stored in the database. See our [2025.4 release notes](https://version-2025-4.goauthentik.io/docs/releases/2025.4#sessions-are-now-stored-in-the-database) for more information.

Reviewer: ```suggestion
### ADDENDUM May 8, 2025: As of version 2025.4 the option to store sessions in cache has been removed; sessions are now exclusively stored in the database. See our [2025.4 release notes](https://version-2025-4.goauthentik.io/docs/releases/2025.4#sessions-are-now-stored-in-the-database) for more information.
```

> 

Reviewer: Excellent. :-)

### Summary

When authentik was configured to use the database for session storage (which is a non-default setting), deleting sessions via the Web Interface or the API would not revoke the session and the session holder would continue to have access to authentik.

This also affects automatic session deletion when a user is set to inactive or a user is deleted.

> The session backend was configured via [this](https://version-2025-2.goauthentik.io/docs/install-config/configuration/#authentik_session_storage-authentik-20244) setting; if this settings isn't set the sessions are stored in the cache (Redis), which is not affected by this.

Reviewer: ```suggestion
The session backend was configured via the `AUTHENTIK_SESSION_STORAGE` setting, which was removed in version 2025.4. 
```

> The session backend is configured via [this](https://version-2025-2.goauthentik.io/docs/install-config/configuration/#authentik_session_storage-authentik-20244) setting; if this settings isn't set the sessions are stored in the cache (Redis), which is not affected by this.

Reviewer: Is this necessary? it changes the URL to match the badge, but it works fine without it, as it is now. Why change it?

Author: IDK i just clicked the header and copied the URL

Author: I've tried it out and without the full anchor I'm not brought to the right pla d

Author: Tried it out in the deploy preview and anchors are generated without the version badge. uhhhhhh...
